cmd/ts-info/app/hevc.go

```diff
@@ -46,69 +47,54 @@ func (a *hevcPS) setPPS(nalu []byte) error {
 	return nil
 }
 
-func parseHEVCPES(jp *jsonPrinter, d *astits.DemuxerData, ps *hevcPS, verbose bool) (*hevcPS, error) {
+func parseHEVCPES(jp *jsonPrinter, d *astits.DemuxerData, ps *hevcPS, o Options) (*hevcPS, error) {
 	pid := d.PID
 	pes := d.PES
+	fp := d.FirstPacket
 	if pes.Header.OptionalHeader.PTS == nil {
 		return nil, fmt.Errorf("no PTS in PES")
 	}
-
 	nfd := naluFrameData{
 		PID: pid,
 	}
-	if d.FirstPacket != nil {
-		af := d.FirstPacket.AdaptationField
-		if af != nil {
-			nfd.RAI = af.RandomAccessIndicator
+	if ps == nil {
+		// return empty PS to count picture numbers correctly
```

Comment:
Let us start at the first picture having SPS/PPS instead of the first picture seen.